bubblewrap.c

```diff
@@ -219,6 +220,7 @@ usage (int ecode, FILE *out)
            "    --info-fd FD                 Write information about the running container to FD\n"
            "    --new-session                Create a new terminal session\n"
            "    --die-with-parent            Kills with SIGKILL child process (COMMAND) when bwrap or bwrap's parent dies.\n"
+           "    --no-reaper                  Do not install a reaper process with PID=1\n"
```

Comment:
In general, this option only makes sense with unshare-pid. Maybe we should verify that and print a warning if not. Maybe it could be `--unshare-pid-noinit`?